xml/bp_troubleshooting_cephadm.xml

```diff
@@ -375,5 +375,17 @@ cat config-json.json
 <screen>
 &prompt.cephuser;ceph orch daemon redeploy <replaceable>daemon</replaceable> --image=myregistry.com/<replaceable>myimage</replaceable>:<replaceable>mytag</replaceable>
 </screen>
- </sect1>
+   <important>
+    <para>
+     If you plan to run <command>ceph-salt update</command> as described in
+     <xref linkend="rolling-updates-running"/> after you apply a PTF, it will not
+     affect the manual changes made by the temporary fix.
```

Comment:
```suggestion
    If you plan to run the <command>ceph-salt update</command> command after applying a PTF
    (as described in <xref linkend="rolling-updates-running"/>), it will not
     affect the manual changes made by the temporary fix.